packages/web-components/src/design-system-provider/index.ts

```diff
@@ -77,601 +137,879 @@ export class FluentDesignSystemProvider extends DesignSystemProvider
 
   /**
    * Define design system property attributes
+   * @remarks
+   * HTML attribute: background-color
+   *
+   * CSS custom property: --background-color
    */
-  @designSystemProperty({
-    attribute: 'background-color',
-    default: DesignSystemDefaults.backgroundColor,
-  })
-  public backgroundColor: string;
-  protected backgroundColorChanged(): void {
-    // If background changes or is removed, we need to
-    // re-evaluate whether we should have paint styles applied
-    this.noPaintChanged();
-  }
-
-  @designSystemProperty({
-    attribute: 'neutral-base-color',
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.neutralBaseColor,
-  })
-  public neutralBaseColor: string;
-  protected neutralBaseColorChanged(oldValue: string, newValue: string): void {
-    const color = parseColorHexRGB(newValue);
-    if (color) {
-      this.neutralPalette = createColorPalette(color);
-    }
-  }
-
-  @designSystemProperty({
-    attribute: 'accent-base-color',
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.accentBaseColor,
+  @attr({
+    attribute: 'fill-color',
   })
-  public accentBaseColor: string;
-  protected accentBaseColorChanged(oldValue: string, newValue: string): void {
-    const color = parseColorHexRGB(newValue);
-    if (color) {
-      this.accentPalette = createColorPalette(color);
-    }
-  }
+  @designToken(fillColor)
+  public fillColor: string;
 
-  @designSystemProperty({
-    attribute: false,
-    cssCustomProperty: false,
-    default: DesignSystemDefaults.neutralPalette,
-  })
-  public neutralPalette: string[];
+  /**
+   * Defines the palette that all neutral color recipes are derived from.
+   * This is an array for hexadecimal color strings ordered from light to dark.
+   *
+   * @remarks
+   * HTML attribute: N/A
+   */
+  @observable
+  @designToken(neutralPalette)
+  public neutralPalette: PaletteRGB;
```

Comment:
In relation to our chat yesterday, this will be changing to the base type Palette in the design tokens.

I just took care of the related change in fast-components.